packages/web/views/components/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+interface IButtonProps {
+  children?: React.ReactNode;
+  type: 'button' | 'submit';
+  onClick: () => void;
+}
+
+const StyledButton = styled.button``;
+
+export const Button = ({
```

Comment:
I recently learned that you can do `Button: React.FC<ButtonProps>`. Then you don't have to explicitly state `children` in your props